packages/auth/src/utils/errors.ts

```diff
@@ -0,0 +1,17 @@
+import { CombinedError } from "urql";
+
+/**
+ * Contains auth error
+ *
+ * Determine if one of the errors we received
+ * is related to the user being deleted
+ */
+export const containsUserDeletedError = (
+  combinedError: CombinedError | undefined,
+) => {
+  return combinedError?.graphQLErrors.some((graphQLError) => {
+    return Object.values(graphQLError.extensions).some((extension) => {
+      return ["user_deleted"].includes(String(extension));
```

Comment:
This seems like an odd way to compare strings, why did you write it like this?

Reply:
Yeah, it's a bit weird.  I copied it from https://github.com/GCTC-NTGC/gc-digital-talent/blob/b08a0c7243a777195f38722a3058896983e5cf28/packages/client/src/utils/errors.tsx#L113 since it matches two different strings.  I left it in this form since they were originally beside each other in the same file but I can switch it to a regular string comparison if you prefer.

Reply:
I've gone ahead and switched this to a regular string comparison.
[better string comparison](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7925/commits/cc54b642243fc27907f13b8eae88e5d21c79af70)